app/views/admin/promotional_features/edit.html.erb

```diff
@@ -1,12 +1,32 @@
 <% page_title "Edit #{@promotional_feature.title} promotional feature" %>
-<h1>Edit ‘<%= @promotional_feature.title %>’ feature</h1>
-
+<% content_for :page_title, "Edit #{@promotional_feature.title} promotional feature" %>
+<% content_for :title, "Edit #{@promotional_feature.title} promotional feature" %>
+<% content_for :context, @organisation.name %>
+<% content_for :title_margin_bottom, 6 %>
```

Comment:
We need to add the error summary component here 

```
<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @promotional_feature)) %>

```

before

<img width="469" alt="image" src="https://user-images.githubusercontent.com/42515961/236143731-b79856dc-3acc-4fa3-91f6-b59bc460ddaf.png">

After

<img width="497" alt="image" src="https://user-images.githubusercontent.com/42515961/236143996-a0f8b1bf-76be-481d-82cd-aba1bc30ecae.png">

Reply:
done